tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingTestCase.java

```diff
@@ -302,7 +302,7 @@ public void testBuildAndRunModuleWithPeriod() throws BallerinaTestException {
      */
     private Map<String, String> addEnvVariables(Map<String, String> envVariables) {
         envVariables.put(ProjectDirConstants.HOME_REPO_ENV_KEY, tempHomeDirectory.toString());
-        envVariables.put("BALLERINA_DEV_STAGE_CENTRAL", "true");
```

Comment:
Line 185 `String url = RepoUtils.getStagingURL() + "/modules/info/" + orgName + "/" + moduleName + "/*/";` seems to be still using the Staging URL. Looks like this could be causing the Travis test failure in `testPullCount ` test case.

Reply:
Thanks, I will look in to it